### PR TITLE
Added Contributing File

### PR DESCRIPTION
**Pull request by ColmBhandal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+### Deploy
+
+```bash
+# Create a model
+juju add-model dev
+# Enable DEBUG logging
+juju model-config logging-config="<root>=INFO;unit=DEBUG"
+# Deploy the charm
+juju deploy ./kubeflow-profiles_ubuntu-20.04-amd64.charm \
+    --resource profile-image=$(yq '.resources."profile-image"."upstream-source"' metadata.yaml)
```
Comment (ColmBhandal) [inline CONTRIBUTING.md:60]: @orfeas-k Is the `profile-image=` correct here? Or should it be `oci-image=`?

Comment (orfeas-k) [reply CONTRIBUTING.md:60]: @ColmBhandal This is right but profiles also needs a [kfam image resource  ](https://github.com/canonical/kubeflow-profiles-operator/blob/main/metadata.yaml#L21) which means that you can add a `--resource kfam-image=$(yq '.resources."kfam-image"."upstream-source"' metadata.yaml)`

Comment (ColmBhandal) [reply CONTRIBUTING.md]: Cool thanks

**Review by orfeas-k — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Deploy
+
+```bash
+# Create a model
+juju add-model dev
+# Enable DEBUG logging
+juju model-config logging-config="<root>=INFO;unit=DEBUG"
+# Deploy the charm
+juju deploy ./kubeflow-profiles_ubuntu-20.04-amd64.charm \
```
Comment (orfeas-k) [inline CONTRIBUTING.md:59]:
```suggestion
juju deploy ./kubeflow-profiles_ubuntu-20.04-amd64.charm --trust \
```

**Review by orfeas-k — APPROVED** [review]

**Pull request merged**